pybleau/app/plotting/heatmap_factory.py

```diff
@@ -53,6 +55,15 @@ def _plot_data_single_renderer(self, x_arr=None, y_arr=None, z_arr=None,
         data_map.update(adtl_arrays)
         return data_map
 
+    def compute_label_positions(self, labels):
+        """ Compute the location of the center of the image cells.
+        """
+        num_val = len(labels)
+        axis_high = num_val - 1
+        cell_starts = np.arange(num_val) * axis_high/num_val
```

Comment:
Do we need to put any precautionary measures in here to handle if `num_val` is 0?

Reply:
Good point. Done.